model/trace/gen-ai.yaml

```diff
@@ -1,13 +1,12 @@
 groups:
-  - id: trace.gen_ai.client
+  - id: trace.gen_ai.client.common_attributes
     type: span
     brief: >
       Describes GenAI operation span.
     attributes:
-      - ref: gen_ai.system
-        requirement_level: required
       - ref: gen_ai.request.model
-        requirement_level: required
+        requirement_level:
+          conditionally_required: If available.
```

Comment:
Azure AI Inference allows to use specific model-as-a-service and does not always have a model name at request time.

Client interacts with the endpoint, but does not need to know the model. 

So, changing level to `conditionally_required` to accommodate this scenario. Other system-specific conventions may override it and say it's required

Reply:
Interesting. It used to be that the request would use the "deployment name" which then would have an underlying model configured in the service. Is that still the case?

Reply:
if you use [serverless models](https://learn.microsoft.com/en-us/azure/ai-studio/how-to/deploy-models-serverless?tabs=azure-ai-studio), after you deploy it, the only thing the client knows is the endpoint  🤷‍♀️

```js
const client = ModelClient("https://contoso.eastus2.inference.ai.azure.com", credential);
  const response = await client.path("/chat/completions").post({
    body: {
      messages: [
        ...
      ],
    }
  });
```